ui/src/error.ts

```diff
@@ -16,9 +16,11 @@ export interface Error {
 }
 
 export enum Code {
-  SessionFetchFailure = "SessionFetchFailure",
   ProjectRequestFailure = "ProjectRequestFailure",
+  SessionFetchFailure = "SessionFetchFailure",
   UnexpectedProxyExit = "UnexpectedProxyExit",
```

Comment:
Do you want to do the rename you mentioned in the docs PR now?

Reply:
Will do separately